requirements/pip_tools.txt

```diff
@@ -4,8 +4,10 @@
 #
 #    make upgrade
 #
-click==8.0.0
-    # via pip-tools
+click==7.1.2
+    # via
+    #   -c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt
+    #   pip-tools
```

Comment:
Going down from `8.0.0` to `7.1.2`

- Pinned <8.0.0 in https://github.com/edx/edx-lint/commit/b9df2934ed57e3c0c825217274318cbe2eaa73e3
- Unpinned in https://github.com/edx/edx-lint/commit/2ea5a1c8a98f120fdb67227a6ff4b57f3cce2698. Hopefully a new requirement update will come through putting this back.